api/formatters/covjson.py

```diff
@@ -40,12 +41,11 @@ def make_parameter(ts_mdata):
 
     return Parameter(
         description={
-            "en": f"{ts_mdata.standard_name} at {level}m, "
-            f"aggregated over {period} with method '{ts_mdata.function}'",
+            "en": f"{label} at {level}m, " f"aggregated over {period} with method '{ts_mdata.function}'",
```

Comment:
Make this one string.

Reply:
Fixed in: https://github.com/EURODEO/e-soh/pull/209/commits/167e77552c40480e7aaa3b9453c2e720c24a97a0